client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-action-bar.tsx

```diff
@@ -0,0 +1,77 @@
+import { Icon, reusableBlock, chevronUp, chevronDown, closeSmall } from '@wordpress/icons';
+import classNames from 'classnames';
+
+type PatternActionBarProps = {
+	onReplace: () => void;
+	onDelete: () => void;
+	onOrderUp?: () => void;
+	onOrderDown?: () => void;
+	disableOrderUp?: boolean;
+	disableOrderDown?: boolean;
+	enableOrdering?: boolean;
+};
+
+const PatternActionBar = ( {
+	onReplace,
+	onDelete,
+	onOrderUp,
+	onOrderDown,
+	disableOrderUp,
+	disableOrderDown,
+	enableOrdering,
+}: PatternActionBarProps ) => {
+	return (
+		<div className="pattern-action-bar" role="menubar" aria-label="Pattern Actions">
+			{ enableOrdering && (
+				<div className="pattern-action-bar__block">
+					<div
+						className={ classNames( 'pattern-action-bar__action --action-order-up', {
+							'--disabled': disableOrderUp,
+						} ) }
+						role="menuitem"
+						aria-label="Order Up"
```

Comment:
We need to translate the label and title. Also, I think it might be better to call it `Move up` and `Move down` to align with Gutenberg.

See https://github.com/WordPress/gutenberg/blob/2de41972289df3c423b604a59720bb38e7d92815/packages/block-editor/src/components/block-mover/button.js#L49